packages/core/src/App/Containers/RealityCheckModal/reality-check-modal.jsx

```diff
@@ -43,6 +43,7 @@ const RealityCheckModal = ({
     setRealityCheckDuration,
     setReportsTabIndex,
     setVisibilityRealityCheck,
+    show_reality_check,
```

Comment:
nit: change to `should_show_reality_check` ?

Reply:
@zaki-binary combined it with other method, please re-check it